src/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -113,15 +113,15 @@ private void decodeParameter(Position position, int id, ChannelBuffer buf, int l
             case 19:
                 position.set("axisZ", readValue(buf, length, true));
                 break;
+            case 21:
```

Comment:
It is 1-5 range, not RSSI 0-31. As an option we can multiply it to fit RSSI range and store as RSSI